pulp_rpm/app/models/package.py

```diff
@@ -29,6 +29,30 @@ def db_type(self, connection):
         return "pulp_evr_t"
 
 
+def format_nevra(name=None, epoch=0, version=None, release=None, arch=None):
```

Comment:
do you think moving this to utils would be a better place?

Reply:
Done